docs/rules.md

```diff
@@ -56,9 +56,10 @@ Please refer to [api-version-parameter-required.md](./api-version-parameter-requ
 
```

Comment:
Looks like there were some odd changes in here, run `rush regen-ruleindex` and commit the updated file.